src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -348,6 +348,36 @@ private void testPublishUnpublishActions() {
         homePage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_PUBLISHED);
         assertTrue(BackDoor.getFeedbackSession(feedbackSessionPublished.getCourseId(),
                                                feedbackSessionPublished.getFeedbackSessionName()).isPublished());
+
+        ______TS("resend links action: PUBLISHED feedback session");
+        homePage.verifyResendPublishedLinksButtonExist(feedbackSessionPublished.getCourseId(),
+                                                       feedbackSessionPublished.getFeedbackSessionName());
+        homePage.clickSessionResultsOptionsCaretElement(feedbackSessionPublished.getCourseId(),
+                                                        feedbackSessionPublished.getFeedbackSessionName());
+        homePage.clickResendPublshedLinksLink(feedbackSessionPublished.getCourseId(),
+                                              feedbackSessionPublished.getFeedbackSessionName());
+        homePage.cancelResendPublshedLinksForm();
+        homePage.clickSessionResultsOptionsCaretElement(feedbackSessionPublished.getCourseId(),
```

Comment:
Perhaps add a comment here and above as these are separate tests under one integrated test